exonum/src/storage/values.rs

```diff
@@ -494,4 +509,21 @@ mod tests {
             );
         }
     }
+
+    #[test]
+    fn decimal_round_trip() {
+        let values = [
+            Decimal::from_str("3.14").unwrap(),
+            Decimal::from_parts(1102470952, 185874565, 1703060790, false, 28),
+            Decimal::new(9497628354687268, 12),
+        ];
+
+        for value in values.iter() {
```

Comment:
side note: I wonder why not clippy asking for replacing this with `for value in &values`